src/filters/truncate.js

```diff
@@ -21,8 +21,12 @@ angular.module('openshiftCommonUI')
       if (useWordBoundary !== false) {
         // Find the last word break, but don't look more than 10 characters back.
         // Make sure we show at least the first 5 characters.
-        var startIndex = Math.max(4, charLimit - 10);
-        var lastSpace = truncated.lastIndexOf(/\s/, startIndex);
+        if (truncated.length > charLimit) {
```

Comment:
Related to the request for a word break going 10 characters back... there is a partially related unit test...

```javascript
line:49     it('should return a string that ends on a word boundary', function() {
```
going to look at adding a related ```expect``` or new test